lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -228,3 +228,39 @@ def test_never_update(monkeypatch, capsys):
     # test framework doesn't see tkinter callback exceptions normally
     # see tkinter.Misc.report_callback_exception
     assert "Exception in Tkinter callback" not in capsys.readouterr().err
+
+
+@pytest.mark.backend('Qt5Agg', skip_on_importerror=True)
+def test_lazy_linux_headless():
+    if sys.platform != "linux":
+        pytest.skip("this a linux-only test")
```

Comment:
Would be more clear as a decorator

```
@pytest.mark.skipif(sys.platform != "linux", reason="linux-only test")
```
Or does this interfere with our `backend` mark?

Reply:
More that I did not remember how to do it decorator style, will change.